app/services/dependabot_service.rb

```diff
@@ -1,159 +1,150 @@
-
 # frozen_string_literal: true
-require 'dependabot/file_fetchers'
-require 'dependabot/file_parsers'
-require 'dependabot/update_checkers'
-require 'dependabot/file_updaters'
-require 'dependabot/pull_request_creator'
-require 'dependabot/omnibus'
-require 'gitlab'
 
 class DependabotService
     # This script is designed to loop through all dependencies in a GHE, GitLab or
     # Azure DevOps project, creating PRs where necessary.
-  def run
-    credentials = [
-      {
-        'type' => 'git_source',
-        'host' => 'gitlab.com',
-        'username' => 'x-access-token',
-        "password" => "randomtoken"
-      }
-    ]
-
-    # Full name of the repo you want to create pull requests for.
-    repo_name = "jithindasad/sample-app-2"  # namespace/project
-
-    # Directory where the base dependency files are.
-    directory = '/'
-
-    # Name of the package manager you'd like to do the update for. Options are:
-    # - bundler
-    # - pip (includes pipenv)
-    # - npm_and_yarn
-    # - maven
-    # - gradle
-    # - cargo
-    # - hex
-    # - composer
-    # - nuget
-    # - dep
-    # - go_modules
-    # - elm
-    # - submodules
-    # - docker
-    # - terraform
-    package_manager = 'bundler'
-    gitlab_hostname = 'gitlab.com'
-
-
-      source = Dependabot::Source.new(
+    @@package_manager = 'bundler'
```

Comment:
Avoid class variables
https://github.com/troessner/reek/blob/master/docs/Class-Variable.md

In this case, constants are a better choice